LondonBritishLibrary/orient/BLorient13266.xml

```diff
@@ -0,0 +1,453 @@
+                                    <layout columns="2" writtenLines="20"><locus from="196" to="201"/></layout>
+                                </layoutDesc>
+                            </objectDesc>
+                            
+                            <handDesc>
+                                <handNote script="Ethiopic" xml:id="h2">
+                                    <desc>Careful handwriting</desc>
+                                    <date notBefore="1700" notAfter="1800" resp="PRS8999Strelcyn"/>
+                                </handNote>
+                            </handDesc>
+                            <additions>
+                                <list>
+                                    <item xml:id="e4">
+                                        <locus from="200r"/>
+                                        <desc type="Comment">Note about the origin of the text.</desc>
+                                        <q xml:lang="gez"><gap reason="ellipsis"/>ለዛቲ፡ መጽሐፍ፡ ከመ፡ ዝ፡ ምጽአታ፡ እስመ፡ ነበረ፡ ፩መኰንን፡ በሀገረ፡ አፍርንጊ፡ ወቦአ፡ ውስተ፡ 
+                                            ጾማእት፡ ከመ፡ ያሰምሮ፡ ለአምላክ፡ <pb n="200v"/>ወሰአሎ፡ ጸሎተ፡ በዘይገብር፡ ወአወረዱ፡ ሎቱ፡ መላእክት፡ ዘንተ፡ መጽሐፈ፡ እምሰማይ። 
+                                            <gap reason="ellipsis"/></q>
+                                    </item>
+                                    <item xml:id="e5">
+                                        <locus target="#201vb"/>
+                                        <desc type="Comment">The translator of the book from the 'European' language is mentioned. The end is missing.
+                                            </desc>
+                                        <q xml:lang="gez"><gap reason="ellipsis"/>ተርጐምክዎ፡ አነ፡ 
+                                            <persName role="translator" ref="PRS14365GabraKrestos">ገብረ፡ ክርስቶስ፡</persName> ለዛቲ፡ መጽሐፍ፡ እምነገረ፡ 
+                                            አፍርንጊ፡ ኀበ፡ ነገረ፡ ኢትዮጵያ። <gap reason="lost"/></q>
+                                    </item>
+                                    <item xml:id="e6">
+                                        <desc type="OwnershipNote">The name of the owner is erased.</desc>
+                                    </item>
+                                </list>
+                            </additions>  
+                        </physDesc>
+                        <history>
+                            <origin>
+                                <origDate notBefore="1700" notAfter="1800">18th century</origDate>
+                            </origin>
+                        </history>
+                        <additional>
+                            <adminInfo>
+                                <recordHist>
+                                    <source>
+                                        <listBibl type="catalogue">
+                                            <bibl>
+                                                <ptr target="bm:Strelcyn1978BritishLibrary"/>
```

Comment:
the bibliography should be for the main part of the description, not for the parts